.changeset/rude-laws-repair.md

```diff
@@ -0,0 +1,6 @@
+---
+"react-live": minor
+"website": patch
```

Comment:
```suggestion
```

Since we're not releasing website via package manager, I don't think we need this here.